test/cppsuite/tests/cursor_bound_01.cpp

```diff
@@ -51,10 +51,27 @@ class cursor_bound_01 : public test {
     struct bound {
```

Comment:
I would make it a class with private members `_key`, `_inclusive` and `_lower_bound`. Then you can use the same variable names without the prefix `_` in the constructor. I think it is good practice to create a class as soon as you need to define functions for it.

Reply:
I tried to do this, and realised that we won't be able to get the value of `_key` from outside the class anymore. Should we utilise a getter function in that case?

Reply:
In a class, members are private by default. You would need to create a getter for each of them or place them into a `public:` section.

Reply:
I think getter makes sense if this is a class, I can go either way on the class / struct debate but if Etienne wants a class I'm happy to back him up.